routes/wpt.js

```diff
@@ -28,9 +28,10 @@ const routeCallback = (error, result, res, rollBarMsg) => {
 const wtp = (app) => {
   app.get('/test/:testId', (req, res) => {
     let testId = req.params.testId;
+    const quality = req.query.quality || 'good';
```

Comment:
Perhaps worth adding some kind of params validation? one which either returns an error on bad input or falls back to the default value?

Reply:
This parameter is coming from a drop-down in the UI so the options are closed there.
if we add server-side validations we would have to maintain them if quality names are added/changed in the cloudinary back end.
if you send an unknown string we return the error we get back from the cloudinary backend.

`
    "error": {
        "http_code": 400,
        "message": "Eager Invalid 'quality' param: 'unknown'., Eager Invalid 'quality' param: 'unknown'., Eager Invalid 'quality' param: 'unknown'., Eager Invalid 'quality' param: 'unknown'., Eager Invalid 'quality' param: 'unknown'., Eager Invalid 'quality' param: 'unknown'."
    },
    "logLevel": "error",
    "message": "Error getting results from cloudinary",
    "status": "error"
}
`
so I think it is not nececery.